### ISSUE TITLE
Calling _gc too much causes the application to crash

### ISSUE THREAD

**Issue by Keithcat1** [jart/cosmopolitan#266]
I was playing around with _gc and _defer and wanted to test if it was noticeably slow and my binary started crashing.
Calling _defer or _gc more than approximately 68074 times causes the binary to crash. It feels like it's running out of stack space or something.
My example is:
void main() {
	_defer(printf, "Done!\n");
	for(uLong i=0; i<100000; i++) {
		printf("%i ", i);
		_defer(free, malloc(100)); // same behavior if _gc() is used
	}
	printf("Say something\n");
	getchar();
}

This doesn't happen if I only make the loop run 10000 times.
Tested on windows 10 using cross-9 Windows to Linux GCC cross-compiler

**Comment by jart** [comment]
I've been meaning to make mappings unlimited on NT. Thanks for reporting you were impacted. It helps a lot with triage. This is now fixed. New [nightlies](https://justine.lol/cosmopolitan/download.html) on the website. @pkulchenko unconfirmed but I might have fixed the NT fork() bug too.

Concerning the performance of _gc(malloc()) vs. free(malloc()), the GC version goes 3x slower (20 nanosecond overhead) but it's sooo worth it. Here's the microbenchmark:

```sh
m=; make -j8 -O MODE=$m o/$m/test/libc/nexgen32e/gclongjmp_test.com.runs TESTARGS=-b
gc(malloc(16))             l:        89𝑐        29𝑛𝑠   m:       124𝑐        40𝑛𝑠
free(malloc(16))           l:        32𝑐        10𝑛𝑠   m:        77𝑐        25𝑛𝑠
```

**Comment by jart** [comment]
Fixed in 34b68f1945b6f5452515957335a579ffa72daff6
